source/docs/fedora_atomic_bare_metal_installation.md

```diff
@@ -66,9 +66,13 @@ Fedora 25 Atomic:
 
     # NOTICE: This will download the content from upstream; this will be very slow.
     # Create your own OSTree repo locally and mirror the content instead.
-    ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=https://dl.fedoraproject.org/pub/fedora/linux/atomic/25/ --ref=fedora-atomic/f25/x86_64/docker-host
+    ostreesetup --nogpg --osname="fedora-atomic" --remote="fedora-atomic-26" --url="https://kojipkgs.fedoraproject.org/atomic/26" --ref="fedora/26/x86_64/atomic-host"
 
-There are several other options you can specify in the kickstart file, see *Kickstart Options* in the [Fedora Installation guide](https://docs.fedoraproject.org/en-US/Fedora/24/html/Installation_Guide/chap-kickstart-installations.html), or this [blog post](/blog/2016/10/install-with-kickstart/) which shows a more complex configuration with disk partitioning.
+    # Alternately, install from the ISO media:
+    # ostreesetup --osname="fedora-atomic" --remote="fedora-atomic-26" --url="file:///ostree/repo" --ref="fedora/26/x86_64/atomic-host" --nogpg
+    # however, you will need to reset ostree setup afterwards in %post
+
+There are several other options you can specify in the kickstart file, see *Kickstart Options* in the [Fedora Installation guide](https://docs.fedoraproject.org/en-US/Fedora/24/html/Installation_Guide/chap-kickstart-installations.html), or this [blog post](/blog/2017/07/fedora-atomic-26-kickstarts) which shows a more complex configuration with disk partitioning.
 
 After creating the configuration file, you have to ensure that it will be available during the installation. Place the file on hard drive, network, or removable media as described in the [Fedora Installation guide](https://docs.fedoraproject.org/en-US/Fedora/24/html/Installation_Guide/chap-kickstart-installations.html).  As a simple example, if the kickstart file is on another machine in the same network, you could serve it using a simple HTTP server:
 
```

Comment:
might want to add a note about poking a hole in your firewall if that port isn't open